VEDA_OS01/tests/utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Utils for tests.
+"""
+from VEDA_OS01.models import TranscriptCredentials
+
+
+def invalidate_fernet_cached_properties():
+    """
+    Invalidates transcript credential fernet field's cached properties.
+    """
+
+    def invalidate_fernet_cached_property(field_name):
+        """
+        Invalidates fernet fields cached properties.
+        """
+        field = TranscriptCredentials._meta.get_field(field_name)
+
+        if field.keys:
+            del field.keys
```

Comment:
We will get `AttributeError`, if `field` has not been accessed before i.e. not cached. Since, we are going to use it in management command, lets keep this in our mind.